graphgym/contrib/loader/roland_as.py

```diff
@@ -0,0 +1,166 @@
+"""
```

Comment:
We have merged `roland_xx` to a single file called `roland`, so I think this should be removed?